lib/tokens/token.js

```diff
@@ -41,41 +41,13 @@ module.exports = function (log, config, random, P, hkdf, Bundle, error) {
     this.createdAt = details.createdAt || 0
```

Comment:
Defaulting to Date.now might give us back that bug with tokens that never expire.